src/components/WalletContext.tsx

```diff
@@ -144,7 +147,7 @@ export default function WalletContext() {
                 <p key={i}>
                   <input type="checkbox" id="i" value="i" defaultChecked={true} onChange={() => handleOnChange(i)} />
                   <label htmlFor="i">
-                    {pwSudtCell.sudt.name}, {pwSudtCell.amount.toString()}
+                    {pwSudtCell.sudt.name}, {pwSudtCell.amount.div("1000000000000000000").toString()}
```

Comment:
1. do NOT hard code here, each type of SUDT has its own decimals. We should implement a utility function for displaying the human-readable amount of a SUDT
2. The general user convention is `amount + symbol`, rather than `name: amount`